### PR TITLE
fix: remove currency under feature flag

### PR DESCRIPTION
**Pull request by henrily-ledger**
### 📝 Description

Remove FF for currency that has been already release, hope it will help for currency disappearance 
### ❓ Context

- **Impacted projects**: `live-desktop, live-mobile` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: libs/ledgerjs/packages/types-live/src/feature.ts [hunk 1]
```diff
@@ -13,18 +13,6 @@ export type FeatureId =
...
-  | "currencyMoonbeam"
-  | "currencyCronos"
-  | "currencySongbird"
-  | "currencyFlare"
-  | "currencyNear"
-  | "currencyFantomMobile"
-  | "currencyMoonbeamMobile"
-  | "currencyCronosMobile"
-  | "currencySongbirdMobile"
   | "currencyFlareMobile"
```
Comment (hedi-edelbloute) [inline libs/ledgerjs/packages/types-live/src/feature.ts:16]: We should remove it, no ? :D

File: libs/ledgerjs/packages/types-live/README.md [hunk 2]
```diff
@@ -1482,958 +1482,3 @@ Type: {errors: Record<[string](https://developer.mozilla.org/docs/Web/JavaScript
...
...atingsPrompt"` | `"counterValue"` | `"deviceLocalization"` | `"deviceInitialApps"` | `"buyDeviceFromLive"` | `"ptxSmartRouting"` | `"currencyAvalancheCChain"` | `"currencyOsmosis"` | `"currencyOsmosisMobile"` | `"currencyFantom"` | `"currencyMoonbeam"` | `"currencyCronos"` | `"currencySongbird"` | `"currencyFlare"` | `"currencyNear"` | `"currencyFantomMobile"` | `"currencyMoonbeamMobile"` | `"currencyCronosMobile"` | `"currencySongbirdMobile"` | `"currencyFlareMobile"` | `"ptxSmartRoutingMobile"` | `"mockFeature"` | `"multibuyNavigation"` | `"syncOnboarding"` | `"walletConnectLiveApp"` | `"walletConnectEntryPoint"` | `"customImage"` | `"referralProgramDiscoverCard"` | `"referralProgramDesktopBanner"` | `"disableNftSend"` | `"disableNftLedgerMarket"` | `"disableNftRaribleOpensea"` | `"walletNftGallery"` | `"receiveStakingFlowConfigDesktop"` | `"ethStakingProviders"` | `"storyly"` | `"staxWelcomeScreen"` | `"postOnboardingClaimNft"` | `"postOnboardingAssetsTransfer"` | `"firebaseEnvironmentReadOnly"` | `"protectServicesMobile"` | `"swapShowDexQuotes"` | `"stakePrograms"` | `"portfolioExchangeBanner"` | `"objkt"` | `"stakeAccountBanner"` | `"newsfeedPage"` | `"domainInputResolution"`)
```
Comment (hedi-edelbloute) [inline libs/ledgerjs/packages/types-live/README.md:1486]: What happened on this file ?

File: libs/ledgerjs/packages/types-live/src/feature.ts (hunk 1 above)
Comment (hedi-edelbloute) [reply libs/ledgerjs/packages/types-live/src/feature.ts]: Discussed in mp, fixed

**Review by hedi-edelbloute — APPROVED** [review]
LGTM, thanks henri :+1:

**Review by haammar-ledger — APPROVED** [review]

**Review by haammar-ledger — COMMENTED** [review]
Additional non-blocking comments
- Note that we don't need separate changesets for LLD & LLM anymore
- Please link the related JIRA ticket (even if not directly fixing)
- Ideally we should justify why this PR has no associated tests

**Pull request merged**